tests/zfs-tests/tests/functional/rsend/send_realloc_encrypted_files.ksh

```diff
@@ -12,9 +12,11 @@
 #
 
 #
-# Copyright (c) 2019 by Lawrence Livermore National Security, LLC.
+# Copyright (c) 2019, Lawrence Livermore National Security LLC. All rights reserved.
```

Comment:
nit: same as above over 79 characters.

Reply:
Removed "All rights reserved'.
Needs squash later (not at main workstation atm)